pkg/api/conversion_test.go

```diff
@@ -0,0 +1,140 @@
+	crdbackups  = `{"apiVersion":"apiextensions.k8s.io/v1beta1","kind":"CustomResourceDefinition","metadata":{"name":"etcdbackups.etcd.database.coreos.com"},"spec":{"group":"etcd.database.coreos.com","names":{"kind":"EtcdBackup","listKind":"EtcdBackupList","plural":"etcdbackups","singular":"etcdbackup"},"scope":"Namespaced","version":"v1beta2"}}`
+	crdclusters = `{"apiVersion":"apiextensions.k8s.io/v1beta1","kind":"CustomResourceDefinition","metadata":{"name":"etcdclusters.etcd.database.coreos.com"},"spec":{"group":"etcd.database.coreos.com","names":{"kind":"EtcdCluster","listKind":"EtcdClusterList","plural":"etcdclusters","shortNames":["etcdclus","etcd"],"singular":"etcdcluster"},"scope":"Namespaced","version":"v1beta2"}}`
+	crdrestores = `{"apiVersion":"apiextensions.k8s.io/v1beta1","kind":"CustomResourceDefinition","metadata":{"name":"etcdrestores.etcd.database.coreos.com"},"spec":{"group":"etcd.database.coreos.com","names":{"kind":"EtcdRestore","listKind":"EtcdRestoreList","plural":"etcdrestores","singular":"etcdrestore"},"scope":"Namespaced","version":"v1beta2"}}`
+)
+
+func testModelBundle() model.Bundle {
+	b := model.Bundle{
+		Name:     "etcdoperator.v0.9.4",
+		Image:    "quay.io/operatorhubio/etcd:v0.9.4",
+		Replaces: "etcdoperator.v0.9.2",
+		Skips:    nil,
+		Properties: []property.Property{
+			property.MustBuildChannel("singlenamespace-alpha", "etcdoperator.v0.9.2"),
+			property.MustBuildPackage("etcd", "0.9.4"),
+			property.MustBuildPackageRequired("test", ">=1.2.3 <2.0.0-0"),
+			property.MustBuildGVKRequired("testapi.coreos.com", "v1", "Testapi"),
+			property.MustBuildGVK("etcd.database.coreos.com", "v1beta2", "EtcdBackup"),
```

Comment:
But here it gets trimmed down to `etcd.database.coreos.com` 

Is it supposed to have the `etcdbackups` prefix? Or am I looking at the wrong place?

Reply:
Wrong place. This is the group portion of the GVK.

Reply:
Ah, makes sense